### ISSUE TITLE
arista.cvp does not support Ansible version 2.17.3

### ISSUE THREAD

**Issue by VincentBP2024** [aristanetworks/ansible-cvp#717]
### Issue Summary

I download the AVD and the latest Ansible for demo with CVP. However, I got the message "arista.cvp does not support Ansible version 2.17.3" once I run the ansible-playbook build.yml. Can someone advise how to fix it?  

### Which component(s) of AVD impacted

other

### How do you run AVD ?

Ansible CLI with AVD Runner

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ClausHolbechArista** [comment]
Please confirm that you have updated the `arista.cvp` collection to version 3.11.0.

You can see it by running `ansible-galaxy collection list` from the directory where your `ansible.cfg` file is placed.

**Comment by VincentBP2024** [comment]
@ClausHolbechArista You're right! I'm using the older version along with AVD. Once I updated with 3.11.0, the issue was fixed. Thank you.
